product_docs/docs/migration_toolkit/55/07_invoking_mtk/mtk_command_options_in_file/executing_migration_with_txt.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: "Executing a migration with a command options file"
+navTitle: "Executing a migration with the options file"
+deepToC: true
+---
+
+After you have created the options file, execute the migration command by referencing the options file:
+
+```shell
+./runMTK.sh -optionsFile <example.options>
+```
+
+!!!note
+   Provide the full path for `<example.options>` if you are running the MTK command from a different folder than where you have stored the file.
+!!!
+
+You can specify all options in the file, or use a mix of command line and options file to specify the migration parameters.
+
+Here are some examples for specifying all options in the file:
+
+## Example 1
+
+Content of the `<example.options>` text file:
+
+```
+tables comp_schema.emp,comp_schema.dept,finance_schema.acctg
+```
+
+Syntax of the migration command:
+
+```shell
+./runMTK.sh -optionsFile example.options schema_name
+```
+
+Command line equivalent:
+
+```shell
+./runMTK.sh -tables comp_schema.emp,comp_schema.dept,finance_schema.acctg schema_name
+```
+
+## Example 2
+
+Content of the `<example.options>` text file:
+
+```
+allTables
+excludeTables comp_schema.emp,finance_schema.jobhist
+funcs finance_schema.add_two_numbers
+safeMode
+connRetryCount 7
+```
+
+Syntax of the migration command:
+
+```shell
+./runMTK.sh -optionsFile example.options schema_name
+```
+
+Command line equivalent:
+
+```shell
+-allTables -excludeTables comp_schema.emp,finance_schema.jobhist -funcs finance_schema.add_two_numbers -safeMode -connRetryCount 7  schema_name
+```
+
+## Example 3
+
+Run offline migration with all schemas
+
+Content of the `<example.options>` text file:
+
+```
+offlineMigration file_dest
+allTables
+schemaOnly
+```
+
+Syntax of the migration command:
+
+```shell
+./runMTK.sh -optionsFile example.options -allSchemas
+```
+
+Command line equivalent:
+
+```shell
+-offlineMigration file_dest -allTables -schemaOnly -allSchemas
```

Comment:
I suggest adding `./runMTK.sh ` to the start of the example shell.

In other words, 
`./runMTK.sh -offlineMigration file_dest -allTables -schemaOnly -allSchemas`